vtex-apps/place-components
# CHK-247: Fix unit test environment

#### What problem is this solving?

The unit tests weren't working when running a `yarn test`. Although almost all tests are still broken, now we are _able to_ fix them.

#### How should this be manually tested?

1. Open this branch with `git fetch origin fix/tests && git switch fix/tests`
2. Run `cd react && yarn && yarn test`
3. `4` tests out of `4` should fail

#### How does this PR make you feel?



_When they ask if the tests are now working_

<img width="300" src="https://user-images.githubusercontent.com/26108090/89431264-103e6b80-d716-11ea-86ec-556d98dbddb0.png" />
